packages/components/src/ListItem/ListItemIcon.tsx

```diff
@@ -57,29 +57,31 @@ export const ListItemIcon: StyledComponent<
 > = styled.div.attrs<ListItemIconProps>(
   ({ color, disabled, density, ...props }) => {
     const theme = useContext(ThemeContext)
-    const { iconGap, iconSize } = listItemDimensions(
+    const { height, iconGap, iconSize, py } = listItemDimensions(
       density || theme.defaults.density
     )
 
     return {
       ...props,
       color: listItemIconColor(color, disabled),
+      density,
+      height,
       iconGap,
       iconSize,
+      py,
     }
   }
 )<ListItemIconInternalProps>`
   align-self: ${({ alignStart }) => (alignStart ? 'flex-start' : 'center')};
   display: flex;
   margin-right: ${({ iconGap, theme }) => theme.space[iconGap]};
-  ${colorHelper}
+  ${({ density, theme }) => listItemPaddingY(density || theme.defaults.density)}
 
-  & > svg,
-  ${StyledIconBase}, ${IconPlaceholder} {
+  ${colorHelper} & > svg,
+    ${StyledIconBase}, ${IconPlaceholder} {
```

Comment:
I think this is a mistake?

```suggestion
  ${colorHelper} 
  
  & > svg,
    ${StyledIconBase}, ${IconPlaceholder} {
```

Reply:
Definitely. Good catch!